src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -39,7 +39,7 @@ internal static class MetricItemExtensions
         internal static void AddBatch(
             this OtlpCollector.ExportMetricsServiceRequest request,
             OtlpResource.Resource processResource,
-            in Batch<Metric> batch)
+            in IEnumerable<Metric> batch)
```

Comment:
Maybe change the variable name to `metrics` to align with the base class https://github.com/open-telemetry/opentelemetry-dotnet/blob/8e1a72b03cd1abf62493bafc589274c99870abf0/src/OpenTelemetry/Metrics/MetricExporter.cs#L29.